bears/ruby/RuboCopBear.py

```diff
@@ -19,7 +19,7 @@ class RuboCopBear:
     """
 
     LANGUAGES = {'Ruby'}
-    REQUIREMENTS = {GemRequirement('rubocop', '0.47.1'),
+    REQUIREMENTS = {GemRequirement('rubocop'),
```

Comment:
though I would recommend to fix it currently for 0.48, and upgrade to newer releases explicitly with an issue. But up to you :+1: Not sure actually what our requirements strategy is for bears^^